README.md

```diff
@@ -20,17 +20,21 @@ know the endpoints for the Channel and EPG data.
 {
     "version": 1,
     "channels": "plugin://plugin.video.example/iptv/channels?output=$FILE",
-    "epg": "plugin://plugin.video.example/iptv/epg?output=$FILE"
+    "channels_format": "json",
```

Comment:
Can we not change `channels` into `channels_url`? So we can also add the (static) channels listing in this file as `channels`?

That would leave us with just one file for dynamic _and_ static use-cases.

Reply:
That would use two keys then, this causes more confusion when both keys are specified to know what has priority. You could use the static listing in 'channels' fine, or am I missing something?

Reply:
We do the check here: https://github.com/add-ons/service.iptv.manager/blob/763657b57145c1e28a3b52923488ce427b1694ca/resources/lib/modules/addon.py#L118

If it's not an url (plugin or http), we try it as a file.

Reply:
I didn't know there was a shortage on JSON keys :upside_down_face: 

It is generally frowned upon in JSON (and any schemaless format for that matter) to have a value that can be both a string and a dict. Another possibility is:
```json
{
  "channels": {
    "format": "json",
    "listing": {
      ...
    },
    "url": "plugin://plugin.video.example/iptv/channels?output=$FILE"
  },
  "epg": {
    "format": "json",
    "url": "plugin://plugin.video.example/iptv/epg?output=$FILE"
  },
  "version": 1,
}
```

I also think we should make an _official_ standard for our JSON format for channels and epg, and give it a name (jsontv and jsonepg?). If anything already exists, we could collaborate, but I haven't found anything conclusive.

Reply:
Hmm, I guess wrt. implementation, leaving the listing out of this JSON makes more sense. I updated my example above. And the Design wiki page.

We should support `special://`, but supporting `file:///` seems useless.

Reply:
Can we not detect the format instead?

I would still prefer to use *settings.xml* so we don't have to access these files from add-on directories. I would expect this to be cached by Kodi and therefore faster than individual file accesses. If not, we have to make sure we get an exception from Kodi devs first.